test/boost/sstable_3_x_test.cc

```diff
@@ -3023,20 +3026,18 @@ static flat_mutation_reader_v2 compacted_sstable_reader(test_env& env, schema_pt
     cf->mark_ready_for_writes();
     lw_shared_ptr<replica::memtable> mt = make_lw_shared<replica::memtable>(s);
 
-    tmpdir tmp;
     auto sstables = open_sstables(env, s, format("test/resource/sstables/3.x/uncompressed/{}", table_name), generations);
     auto new_generation = generations.back() + 1;
 
     auto desc = sstables::compaction_descriptor(std::move(sstables), default_priority_class());
-    desc.creator = [s, &tmp, &env, new_generation] (shard_id dummy) {
-        return env.make_sstable(s, tmp.path().string(), new_generation,
-                         sstables::sstable_version_types::mc, sstable::format_types::big, 4096);
+    desc.creator = [s, &env, new_generation] (shard_id dummy) {
+        return env.make_sstable(s, new_generation);
```

Comment:
was: sstable_version_types::mc
now: (defaulted) get_highest_sstable_version() -> me

Reply:
ditto as for use of `md` above

Reply:
patch extracted in next version